server/docker-compose.yaml

```diff
@@ -0,0 +1,32 @@
+version: "3"
+services:
+  server:
+    image: node:12.18.4
+    ports:
+      - 3000:3000
+    environment:
+      - PGUSER=samza
+      - PGHOST=db
+      - PGPASSWORD=postpass
+      - PGDATABASE=post
+      - PGPORT=5432
+    depends_on:
+      - db
```

Comment:
Why did you set the dependency? Would you not rather have the server run even if the database has some errors?